### PR TITLE
VACMS-17404: PACT Act update links to va-link-action 

### PR DESCRIPTION
**Pull request by chriskim2311**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
For this PR we convert action links in the PACT Act wizard over to the DST va-link-action component. 

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17404

## Testing done
Tested links locally and review markup

## Screenshots
Learn_how_the_PACT_Act_may_affect_you___PACT_Act___Veterans_Affairs

You_may_be_eligible_for_VA_benefits___PACT_Act___Veterans_Affairs

Apply_for_VA_benefits_now___PACT_Act___Veterans_Affairs

## What areas of the site does it impact?
PACT Act Wizard

## Acceptance criteria
- [x] action link is componentized
- [x] other features display correctly
- [x] verify/review the E2E tests
- [x] check desktop and mobile
- [ ] a11y review

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

**Review by randimays — DISMISSED** [review]
LGTM!

**Comment by laflannery** [comment]
@chriskim2311 Here are the VADS class updates [I mentioned in Slack](https://dsva.slack.com/archives/C52CL1PKQ/p1720804894836739)

- Intro link "Get Started":
  - remove `vads-u-display--block`
  - update `vads-u-margin-top--4` to `vads-u-margin-top--2` (this will keep the design the same)

File: src/applications/pact-act/containers/HomePage.jsx [hunk 1]
```diff
@@ -34,14 +34,13 @@ const HomePage = ({ router, setIntroPageViewed }) => {
         to apply.
       </p>
       {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-      <a
+      <va-link-action
         data-testid="paw-start-form"
         href="#"
-        className="vads-u-display--block vads-u-margin-top--4 vads-c-action-link--green"
+        class="vads-u-margin-top--2"
```
Comment (eselkin) [inline src/applications/pact-act/containers/HomePage.jsx:40]: Interesting. Does className not work for non-react components?

**Review by eselkin — APPROVED** [review]
LGTM

**Comment by laflannery** [comment]
Looks good!

**Review by randimays — APPROVED** [review]

File: src/applications/pact-act/containers/HomePage.jsx (hunk 1 above)
Comment (chriskim2311) [reply src/applications/pact-act/containers/HomePage.jsx:40]: @eselkin For the DST components they take class as a prop for className to apply them properly. Thanks for the review!

**Pull request merged**